docs/3.0.0-beta.x/admin-panel/deploy.md

```diff
@@ -28,7 +28,7 @@ You might want to change the path to access to the administration panel. Here th
 }
 ```
 
-**You have to rebuild the administration panel to make this work.** [Build instructions](#build).
+**You have to rebuild the administration panel to make this work.** [Build instructions](https://strapi.io/documentation/3.0.0-beta.x/admin-panel/customization.html#build).
```

Comment:
You should be using relative links with .md files. Vuepress builds the final urls itself